traffic_monitor/experimental/traffic_monitor/static/index.html

```diff
@@ -382,7 +386,7 @@
 
 					 // \todo check that array has a member before dereferencing [0]
 					 if (jds[deliveryService].hasOwnProperty("isAvailable")) {
-						 document.getElementById("deliveryservice-stats-" + deliveryService + "-status").textContent = jds[deliveryService]["isAvailable"][0].value ? "available" : "unavailable";
+						 document.getElementById("deliveryservice-stats-" + deliveryService + "-status").textContent = jds[deliveryService]["isAvailable"][0].value == "true" ? " available" : "unavailable - " + jds[deliveryService]["error-string"][0].value;
```

Comment:
Is the space before " available" intentional? If so, should "unavailable" also have a space before it?